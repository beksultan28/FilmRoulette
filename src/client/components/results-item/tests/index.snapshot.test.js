import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

import ResultsItemComponent from '../index.jsx';

const props = {
  searchBy: 'movie',
  item: {
    id: 1,
    title: 'Test movie',
    vote_average: 5,
    release_date: '2017-01-01',
  },
};
const resultsItem = renderer.create(
  <MemoryRouter>
    <ResultsItemComponent
      {...props}
    />
  </MemoryRouter>,
);

test('ResultsItemComponent renders correctl', () => {
  expect(resultsItem.toJSON()).toMatchSnapshot();
});

