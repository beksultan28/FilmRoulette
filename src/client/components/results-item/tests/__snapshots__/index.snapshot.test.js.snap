// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsItemComponent renders correctl 1`] = `
<div
  className="result-item"
>
  <a
    className="result-content"
    href="/film/movie/1"
    onClick={[Function]}
  >
    <div
      className="result-img"
    >
      <img
        alt=""
        src="http://image.tmdb.org/t/p/w342undefined"
      />
    </div>
    <div
      className="result-title"
    >
      Test movie
    </div>
    <div
      className="clearfix"
    >
      <small
        className="label label-default pull-left"
      >
        IMDB: 
        5
      </small>
      <span
        className="label label-default pull-right"
      >
        2017-01-01
      </span>
    </div>
  </a>
</div>
`;
